boot/environ/CMakeLists.txt

```diff
@@ -85,7 +85,7 @@ if(MSVC)
     if(ARCH STREQUAL "arm")
         add_target_link_flags(bootmgfw "/ignore:4078 /ignore:4254 /DRIVER")
     else()
-        add_target_link_flags(bootmgfw "/ignore:4078 /ignore:4254 /DRIVER /FIXED")
+        add_target_link_flags(bootmgfw "/ignore:4078 /ignore:4254 /DRIVER /DYNAMICBASE:NO /FIXED")
```

Comment:
Why doesn't dynamicbase work when compiling with the IDE only?

Reply:
Visual uses yes as a default value, I guess. Same as with this /MD stuff.